splunk_add_on_ucc_framework/global_config.py

```diff
@@ -60,19 +60,26 @@ def from_dict(cls, **kwargs: Any) -> "OSDependentLibraryConfig":
 
 
 class GlobalConfig:
-    def __init__(self, global_config_path: str) -> None:
+    def __init__(
+        self,
+        content: Dict[str, Any],
+        is_yaml: bool,
```

Comment:
I think we could determine this from `original_path` parameter itself, it seems to be redundant.

Reply:
It might be but the purpose of this PR is not to alter the behavior but to simply refactor a method.

I'd happily make changes to the logic in the next PRs.